### PR TITLE
dev -> staging

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ click==6.7
 coreapi==2.3.3
 coreschema==0.0.4
 coverage==4.5.1
-cryptography==2.2.2
+cryptography==2.1.4
```
Comment (burgwyn) [inline requirements.txt]: Any reason why we would go backwards?

File: talentmap_api/fsbid/services/available_positions.py [hunk 2]
```diff
@@ -99,6 +99,14 @@ def get_available_positions_csv(query, jwt_token, host=None):
     ])
 
     for record in data:
+        try:
```
Comment (burgwyn) [inline talentmap_api/fsbid/services/available_positions.py]: 👍

File: talentmap_api/fsbid/services/employee.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 from django.conf import settings
 from django.contrib.auth.models import Group
 
-API_ROOT = settings.FSBID_API_URL
+API_ROOT = settings.EMPLOYEES_API_URL
```
Comment (burgwyn) [inline talentmap_api/fsbid/services/employee.py]: 👍

File: requirements.txt (hunk 1 above)
Comment (mjoyce91) [reply requirements.txt]: @burgwyn this was the culprit of our ADFS integration breaking

**Pull request merged**